### ISSUE TITLE
To normalize or not to normalize

### ISSUE THREAD

**Issue by alankbi** [alankbi/detecto#16]
On the pre-trained model (without any finetuning or custom training), the accuracy is terrible if images passed into the model are normalized. This will probably require some investigation as to how to properly apply normalization to images, if you're supposed to at all. 

The temporary workaround right now is to disable normalizing when just using the default weights.

**Comment by alankbi** [comment]
Would you be able to provide more specifics about what your error is? As for the normalization, it does seem like there's some built-in normalization going on. With the default model, normalizing twice causes very poor results (which is why I have it disabled), but it looks about the same when using a custom-trained model.

**Comment by dereklukacs** [comment]
I don't have an error anymore, was just explaining how I stumbled on that code. Saw this open issue and wanted to make sure you were aware that the model has normalization built in.
